parsl/dataflow/dflow.py

```diff
@@ -46,7 +45,7 @@ class DataFlowKernel(object):
     """
 
     def __init__(self, config=None, executors=None, lazyErrors=True, appCache=True,
-                 rundir=None, retries=0, checkpointFiles=None):
+                 rundir=None, retries=0, checkpointFiles=None, checkpointMethod=None):
```

Comment:
can we call this `checkpointMode` instead of `checkpointMethod`? This gives 'method' a double meaning so it is confusing.